model/notification/notification.go

```diff
@@ -276,7 +276,7 @@ func CollectUnsentNotificationStats() (*NotificationStats, error) {
 		{
 			"$match": bson.M{
 				sentAtKey: bson.M{
-					"$eq": time.Time{},
+					"$exists": false,
```

Comment:
this assumes we don't mind skewing stats for the brief period after the deploy for the notifications generated before the deploy that haven't been sent yet.

The alternative is to do it in stages: first `{$or: { $exists: false, $eq:time.Time{} }}` and the next day without the $or